### ISSUE TITLE
Download button does not work

### ISSUE THREAD

**Issue by cweiske** [tobexyz/yaacc-code#142]
When audio files are listed in the content view, each file has four buttons:

1. Add to playlist
2. Download
3. Play
4. Play this and all following

The download button does not work here.
There is no notification that anything happens, and nothing relevant in the logs.

(Even if it would work, it'd be nice if I could disable it - when using YAACC as music player, downloading is not among the relevant actions)

----

This is with YAACC debug build 1875a59ea9ad01ec01e2c0b9ba202d37e90ff512 on a Fairphone 4, Android 14 (LineageOS 21) with Gerbera as UPnP server. The UPnP server offers the native .ogg files and a on-demand mp3 conversion as file format options.

**Comment by tobexyz** [comment]
I can't confirm this behaviour. When I push the download button a short notification appears during the download. 
As result the file is located in the folder Downloads/yaacc
If it is not the same on your device please let me know.

Maybe I should add a toast notification...

The feature is part of the app for offline usage of media files.

**Comment by cweiske** [comment]
You are right - the files are downloaded into the `Download/yaacc/` directory.
But on my system, no download notification appears  - maybe it's to fast.

A part of the problem is that the UI buttons in YAACC are static - they do not give feedback when they are/got pressed.

A toast would help indeed.

**Comment by cweiske** [comment]
A "open download folder" in the settings would make the download folder location more visible.

**Comment by tobexyz** [comment]
in version 4.3.2 a toast shows the download location
